### PR TITLE
LPS-142440 Enable edition of selected node info

### PR DESCRIPTION
**Pull request by naraandrade**
Story: [LPS-139387](https://issues.liferay.com/browse/LPS-139387)

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by ramonmsp** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/Sidebar.js [hunk 1]
```diff
@@ -9,31 +9,218 @@
...
+			id: false,
+			label: false,
+		});
+	}, [selectedNode?.id, setSelectedNodeNewId]);
+
 	return (
 		<div className="sidebar">
 			<div className="sidebar-header">
-				<span className="title">{Liferay.Language.get('nodes')}</span>
+				{selectedNode && (
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/Sidebar.js:49]: Hey @naraandrade, Can we break this component into smaller ones? What do you think?

Comment (naraandrade) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/Sidebar.js:49]: Yes, I can create a SidebarHeader component to control this visualization, as I did with the SidebarPanel. Thx :)

Comment (rafaprax) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/Sidebar.js:49]: If you see that this approach could be used in other parts of this file, go ahead. Thx

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:stop

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by jotascript** [comment]
Just started reviewing :)

**Review by jotascript — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:stop

**Comment by ramonmsp** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
ci:stop

**Review by ramonmsp — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**